internal/component/database_observability/mysql/collector/schema_table.go

```diff
@@ -219,6 +234,11 @@ func (c *SchemaTable) extractSchema(ctx context.Context) error {
 		}
 	}
 
+	if len(tables) == 0 {
+		level.Info(c.logger).Log("msg", "no tables detected from information_schema.tables")
```

Comment:
just curious why we want to log this?

Reply:
Found useful for a case where the user was not granted SELECT to that particular schema, so the query on `information_schema.tables` would return an empty result without error (just a blank output). I'm happy to move to debug, maybe?

Reply:
Thanks for the info! I'm ok with it in Info!